moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -343,6 +343,10 @@ class TrajectoryExecutionManager
   bool execution_duration_monitoring_;
   double allowed_execution_duration_scaling_;
```

Comment:
add comment above that these are the 'global' variables, in contrast to the controller specific ones